build/runTests.ps1

```diff
@@ -0,0 +1,127 @@
+function runTests() {
+    Write-Host "Start tests"
+    $x = ""; Get-ChildItem -path . -Recurse -Include *Tests.dll | where { $_.FullName -match "bin" } | foreach { $x += """$_"" " }; iex "& '$env:VSTEST_PATH' /EnableCodeCoverage /Logger:trx $x"
+    testExitCode
+}
+
+#Copied from https://github.com/SonarSource/sonar-csharp/blob/master/scripts/utils.ps1
```

Comment:
The rest of this script is copied from the Sonar-C# repo. I made only one changed which I've called out below.